src/Faker/Provider/nl_NL/Company.php

```diff
@@ -12,4 +12,28 @@ class Company extends \Faker\Provider\Company
     );
 
     protected static $companySuffix = array('VOF', 'CV', 'LLP', 'BV', 'NV', 'IBC', 'CSL', 'EESV', 'SE', 'CV', 'Stichting', '& Zonen', '& Zn');
+
+    /**
+     * Belasting Toegevoegde Waarde (BTW) = VAT
+     *
+     * @example 'NL123456789B01'
+     *
+     * @see (dutch) http://www.belastingdienst.nl/wps/wcm/connect/bldcontentnl/belastingdienst/zakelijk/btw/administratie_bijhouden/btw_nummers_controleren/uw_btw_nummer
+     *
+     *
+     * @return string VAT Number
+     */
+    public static function vat()
```

Comment:
Could you add documentation about these two new formatters in the locale specific section of the Readme?